From 778e8b4dfca1cf6a22d652ed0ea8e999f861b800 Mon Sep 17 00:00:00 2001
From: Sebastian Bauer <mail@sebastianbauer.info>
Date: Sat, 2 Jan 2016 19:42:24 +0100
Subject: [PATCH 8/8] Some clib2 changes.

---
 src/sort.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/sort.c b/src/sort.c
index 3db3ea30ff930cca703c13455e0fd521159ee61f..ffb52bb4fd17d71d1eb5de288e5404be25e7368c 100644
--- a/src/sort.c
+++ b/src/sort.c
@@ -39,15 +39,19 @@
 #include "xstrtol.h"
 
 #if HAVE_SYS_RESOURCE_H
 # include <sys/resource.h>
 #endif
 #ifndef RLIMIT_DATA
+#ifdef RLIM_DATA
+#define RLIMIT_DATA RLIM_DATA
+#else
 struct rlimit { size_t rlim_cur; };
 # define getrlimit(Resource, Rlp) (-1)
 #endif
+#endif
 
 #if defined __amigaos4__ && defined __NEWLIB__
 #undef SA_NOCLDSTOP
 #endif
 
 /* The official name of this program (e.g., no `g' prefix).  */
-- 
2.14.2

