From 6217a02ca0d5bc5789936e683bd1b70bc3ba3420 Mon Sep 17 00:00:00 2001
From: Sebastian Bauer <mail@sebastianbauer.info>
Date: Fri, 6 May 2016 08:40:44 +0200
Subject: [PATCH 9/9] Include emit-rtl.h

---
 gcc/config/rs6000/amigaos.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gcc/config/rs6000/amigaos.c b/gcc/config/rs6000/amigaos.c
index c96903dfbdbdefcc443eb6eb1307b92678565cd3..797261177e46a34ac0aba97f2b9c2b8af8df2e72 100644
--- a/gcc/config/rs6000/amigaos.c
+++ b/gcc/config/rs6000/amigaos.c
@@ -46,12 +46,13 @@
 #include "function.h"
 #include "output.h"
 #include "tm_p.h"
 #include "fold-const.h"
 #include "langhooks.h"
 #include "explow.h"
+#include "emit-rtl.h"
 
 #undef DEBUG
 #ifdef DEBUG
 #define dprintf(...)      			\
 printf("%s: ", __PRETTY_FUNCTION__);		\
 printf(__VA_ARGS__)
-- 
2.1.4

