From 3fbaae9e66b5a962272be8f453b2e128e67a1a28 Mon Sep 17 00:00:00 2001
From: Sebastian Bauer <mail@sebastianbauer.info>
Date: Mon, 30 Jan 2017 20:49:17 +0100
Subject: [PATCH 09/20] Added e500 support for AmigaOS.

---
 gcc/config.gcc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config.gcc b/gcc/config.gcc
index 568f207eed3c16d4b772b47a7d7273de801977e1..ec2f3ce8eae0f4b16f27e954c1575dc84fe8e9ce 100644
--- a/gcc/config.gcc
+++ b/gcc/config.gcc
@@ -2309,13 +2309,13 @@ nvptx-*)
 	;;
 pdp11-*-*)
 	tm_file="${tm_file} newlib-stdint.h"
 	use_gcc_stdint=wrap
 	;;
 powerpc-*-amigaos*)
-	tm_file="${tm_file} dbxelf.h elfos.h rs6000/sysv4.h rs6000/amigaos.h"
+	tm_file="${tm_file} dbxelf.h elfos.h rs6000/sysv4.h rs6000/amigaos.h rs6000/e500.h"
 	tm_p_file="${tm_p_file} rs6000/amigaos-protos.h"
 	extra_options="${extra_options} rs6000/sysv4.opt rs6000/amigaos.opt"
 	tmake_file="rs6000/t-amigaos"
 	extra_objs=amigaos.o
 	use_collect2=no
 	;;
-- 
2.34.1

