From db84bd50e70189be71612ff6baf32371703ce129 Mon Sep 17 00:00:00 2001
From: Sebastian Bauer <mail@sebastianbauer.info>
Date: Tue, 17 Apr 2018 22:02:09 +0200
Subject: [PATCH 16/20] Fix r2 restoring in the epilog of baserel-restoring
 functions.

---
 gcc/config/rs6000/rs6000.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/rs6000/rs6000.c b/gcc/config/rs6000/rs6000.c
index 16c96acdb619e5de889b2fe42f95ec54784f2c6b..84b37268b6bae8529230a22de5bd902452ba5d02 100644
--- a/gcc/config/rs6000/rs6000.c
+++ b/gcc/config/rs6000/rs6000.c
@@ -28403,13 +28403,13 @@ rs6000_emit_epilogue (int sibcall)
 #ifdef TARGET_BASEREL
   if (info->baserel_save_p && TARGET_BASEREL)
     {
       rtx addr, mem, reg;
 
       /* Restore r2 */
-      addr = gen_rtx_PLUS (Pmode, gen_rtx_REG (Pmode, 12),
+      addr = gen_rtx_PLUS (Pmode, frame_reg_rtx,
 			   GEN_INT (info->baserel_save_offset + frame_off)); /* or sp_off? */
       mem = gen_frame_mem (reg_mode, addr);
 
       reg = gen_rtx_REG (reg_mode, 2);
       emit_move_insn (reg, mem);
 
-- 
2.34.1

