? fd2pragma
Index: fd2pragma.c
===================================================================
RCS file: /cvsroot/sditools/sditools/fd2pragma/fd2pragma.c,v
retrieving revision 1.26
diff -u -r1.26 fd2pragma.c
--- fd2pragma.c	25 Mar 2014 19:52:22 -0000	1.26
+++ fd2pragma.c	22 Mar 2015 05:53:07 -0000
@@ -5997,12 +5997,12 @@
     DoOutput("\"r\" (%s)%s", RegNames[j], (i < ap->NumArgs-1 ? ", " : ""));
   }
 
-  DoOutput("\n  : \"d0\", \"d1\", \"a0\", \"a1\", \"fp0\", \"fp1\"");
+  DoOutput("\n  : ");//\"d0\", \"d1\", \"a0\", \"a1\", \"fp0\", \"fp1\"");
 
   if(noret)
-    return DoOutput(/*({*/", \"cc\", \"memory\");\n}\n\n");
+    return DoOutput(/*({*/"\"cc\", \"memory\");\n}\n\n");
   else
-    return DoOutput(/*({*/", \"cc\", \"memory\");\n  return res;\n}\n\n");
+    return DoOutput(/*({*/"\"cc\", \"memory\");\n  return res;\n}\n\n");
 }
 
 
