From caf7e545e372ec8e5412bb73f43d13a7d8591209 Mon Sep 17 00:00:00 2001
From: Sebastian Bauer <mail@sebastianbauer.info>
Date: Sat, 5 Dec 2015 00:32:33 +0100
Subject: [PATCH 7/7] Fixed return type of pex_amiga_exec_child().

---
 libiberty/pex-amigaos.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libiberty/pex-amigaos.c b/libiberty/pex-amigaos.c
index 873ea977f383d19a424c81ef5f8522e6ade18da8..0c61a108764c8501f8a2e9552c7c07499f402b1a 100644
--- a/libiberty/pex-amigaos.c
+++ b/libiberty/pex-amigaos.c
@@ -45,13 +45,13 @@ extern int errno;
 #if !defined(FD_CLOEXEC)
 #define FD_CLOEXEC 1
 #endif
 
 static int pex_amiga_open_read (struct pex_obj *, const char *, int);
 static int pex_amiga_open_write (struct pex_obj *, const char *, int);
-static long pex_amiga_exec_child (struct pex_obj *, int, const char *,
+static pid_t pex_amiga_exec_child (struct pex_obj *, int, const char *,
 				 char * const *, char * const *,
 				 int, int, int, int,
 				 const char **, int *);
 static int pex_amiga_close (struct pex_obj *, int);
 static int pex_amiga_wait (struct pex_obj *, long, int *, struct pex_time *,
 			   int, const char **, int *);
@@ -112,13 +112,13 @@ pex_amiga_close (struct pex_obj *obj ATTRIBUTE_UNUSED, int fd)
 }
 
 /* Execute a child.  */
 
 const unsigned char __shell_escape_character = '\\';
 
-static long
+static pid_t
 pex_amiga_exec_child (struct pex_obj *obj, int flags ATTRIBUTE_UNUSED, const char *executable ATTRIBUTE_UNUSED,
 		     char * const * argv, char * const * env ATTRIBUTE_UNUSED,
                      int in ATTRIBUTE_UNUSED, int out ATTRIBUTE_UNUSED, int errdes ATTRIBUTE_UNUSED,
 		     int toclose ATTRIBUTE_UNUSED, const char **errmsg, int *err)
 {
   int rc;
-- 
2.1.4

