From f6f9ce2214e23fc5964d1d2a2be5563f9d0bbbff Mon Sep 17 00:00:00 2001
From: Sebastian Bauer <mail@sebastianbauer.info>
Date: Thu, 31 May 2018 20:54:41 +0200
Subject: [PATCH 1/2] AmigaOS changes.

---
 m4/ls-mntd-fs.m4 | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/m4/ls-mntd-fs.m4 b/m4/ls-mntd-fs.m4
index a678f05553379d5a9c0adadfd8a72e92b77f8881..0aa1812d6a5e1a01009f5897b6e110261270e89e 100644
--- a/m4/ls-mntd-fs.m4
+++ b/m4/ls-mntd-fs.m4
@@ -345,14 +345,15 @@ if test -z "$ac_list_mounted_fs"; then
     fi
     ;;
   esac
 fi
 
 if test -z "$ac_list_mounted_fs"; then
-  AC_MSG_ERROR([could not determine how to read list of mounted file systems])
+  AC_MSG_WARN([could not determine how to read list of mounted file systems])
   # FIXME -- no need to abort building the whole package
   # Can't build mountlist.c or anything that needs its functions
 fi
 
-AS_IF([test $ac_list_mounted_fs = found], [$1], [$2])
+## HACK TO GET IT TO WORK ON AMIGAOS4
+AS_IF([test $ac_list_mounted_fs = found], [$1], [$1])
 
   ])
-- 
2.17.1

