From 042381d41925b9ae65f87f64bca778a0e16956e7 Mon Sep 17 00:00:00 2001
From: Marcus Comstedt <marcus@mc.pp.se>
Date: Wed, 31 Jul 2019 22:10:48 +0200
Subject: [PATCH 27/27] Default to DWARF v2

---
 gcc/config/rs6000/amigaos.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/gcc/config/rs6000/amigaos.h b/gcc/config/rs6000/amigaos.h
index ec0146c4b8c05eb300f8928e475641123c3f5632..a8724d3f3efdfe759565828190426c180fc3a1ec 100644
--- a/gcc/config/rs6000/amigaos.h
+++ b/gcc/config/rs6000/amigaos.h
@@ -381,12 +381,16 @@ do                                   \
 {                                    \
   if (TARGET_ALTIVEC)                \
   {                                  \
     rs6000_altivec_abi = 1;          \
     TARGET_ALTIVEC_VRSAVE = 1;       \
   }                                  \
+  if (!global_options_set.x_dwarf_strict)	\
+    dwarf_strict = 1;				\
+  if (!global_options_set.x_dwarf_version)	\
+    dwarf_version = 2;				\
 } while(0)
 
 #undef SUBTARGET_EXPAND_BUILTIN
 #define SUBTARGET_EXPAND_BUILTIN(EXP, TARGET, SUBTARGET, MODE, IGNORE, SUCCESS) \
   amigaos_expand_builtin (EXP, TARGET, SUBTARGET, MODE, IGNORE, SUCCESS)
 
-- 
2.30.0

