From 3ce5beb5610ded6b50cac4fd492506ce74f71d0e Mon Sep 17 00:00:00 2001
From: Sebastian Bauer <mail@sebastianbauer.info>
Date: Sun, 8 Jul 2018 15:37:39 +0200
Subject: [PATCH 12/13] Fix pointer comparision bug.

---
 binutils/stabs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/binutils/stabs.c b/binutils/stabs.c
index 1e7e3f2e3aafd4b8dbf4faa7212ed2008ad686de..cd1f2aa200af6a1cab3b68fc9a1295c85547df7e 100644
--- a/binutils/stabs.c
+++ b/binutils/stabs.c
@@ -2694,13 +2694,13 @@ parse_stab_members (void *dhandle, struct stab_handle *info,
 		  bad_stab (orig);
 		  goto fail;
 		}
 	      ++*pp;
 	      voffset &= 0x7fffffff;
 
-	      if (**pp == ';' || *pp == '\0')
+	      if (**pp == ';' || **pp == '\0')
 		{
 		  /* Must be g++ version 1.  */
 		  context = DEBUG_TYPE_NULL;
 		}
 	      else
 		{
-- 
2.30.2

